pkg/security/secl/model/model.go

```diff
@@ -587,6 +587,11 @@ type DNSEvent struct {
 	Response *DNSResponse `field:"response,check:HasResponse"`
 }
 
+// FailedDNSEvent represents a DNS packet that was failed to be decoded (inbound or outbound)
+type FailedDNSEvent struct {
+	Payload []byte `field:"-"`
+}
```

Comment:
Just want to double check: did we really need this type (plus its entry in model_unix.go) ? As it's a custom event and already defined in custom_events.go I wonder.